src/test/java/teammates/test/cases/ui/StudentProfileEditSaveActionTest.java

```diff
@@ -8,12 +8,17 @@
 
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.StudentAttributes;
 import teammates.common.datatransfer.StudentProfileAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
 import teammates.common.util.Const;
 import teammates.common.util.FieldValidator;
 import teammates.common.util.StringHelper;
 import teammates.test.driver.AssertHelper;
 import teammates.ui.controller.RedirectResult;
+import teammates.ui.controller.ShowPageResult;
+import teammates.ui.controller.StudentCourseDetailsPageAction;

```

Comment:
I don't really like having the dependency to `StudentCourseDetailsPageAction` here. This test should only be testing `StudentProfileEditSaveAction` as the test name suggests. What is being tested here and why is this necessary?